### PR TITLE
29 create action and reducer for ass status

### PR DESCRIPTION
**Pull request by Nanitali**
Hi, Create assessment Status reducer and action creators,also write test for action and reducer, please review. Many thanks

### CODE REVIEW

File: client/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export const REQUEST_ASSESSMENT_STATUSES = 'REQUEST_ASSESSMENT_STATUSES'
```
Comment (emilyparkes) [inline client/actions/index.js]: Hi, this looks good but could we rename the file to be `assessmentStatus.js`. Thanks!

File: client/components/App.jsx [hunk 2]
```diff
@@ -1,12 +1,16 @@
 import React from 'react'
-import { BrowserRouter as Router, Switch } from 'react-router-dom'
+
+import Login from './Login'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
```
Comment (emilyparkes) [inline client/components/App.jsx]: Redux changes are not required here. Please remove these changes. They are not part of your ticket.

File: client/components/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from 'react'
```
Comment (emilyparkes) [inline client/components/Login.jsx]: Redux changes are not required here. Please remove these changes. They are not part of your ticket.

File: client/reducers/index.js [hunk 4]
```diff
@@ -3,3 +3,6 @@ import { combineReducers } from 'redux'
 export default combineReducers({
 
 })
+
+
+// Amy will do this
```
Comment (emilyparkes) [inline client/reducers/index.js]: No comments or extra lines please :)

File: server/server.js [hunk 5]
```diff
@@ -3,11 +3,7 @@ const express = require('express')
```
Comment (emilyparkes) [inline server/server.js]: Redux changes are not required here. Please remove these changes. They are not part of your ticket.

File: tests/login.test.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
+const request = require('supertest')
```
Comment (emilyparkes) [inline tests/login.test.js]: Redux changes are not required here. Please remove these changes. They are not part of your ticket.

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: client/reducers/index.js (hunk 4 above)
Comment (struanw) [reply client/reducers/index.js]: All fixed. Thanks

File: tests/login.test.js (hunk 6 above)
Comment (struanw) [reply tests/login.test.js]: removed thanks

File: server/server.js [hunk 7]
```diff
@@ -5,16 +5,30 @@ const cohortsRoute = require('./routes/cohorts')
...
+
 server.use('/api/v1/auth', authRoutes)
 server.use('/cohorts', cohortsRoute)
 
 server.get('*', (req, res) => {
   res.sendFile(path.join(__dirname, 'public/index.html'))
 })
 
+<<<<<<< HEAD
+>>>>>>> 8f9a84037eb5773a290d3b865d54d1faa97e38a6
```
Comment (emilyparkes) [inline server/server.js]: please remove these characters from the conflicts throughout your files.

File: client/reducers/index.js [hunk 8]
```diff
@@ -1,5 +0,0 @@
-import { combineReducers } from 'redux'
-
```
Comment (emilyparkes) [inline client/reducers/index.js]: Please keep this in.

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: client/reducers/index.js (hunk 8 above)
Comment (Nanitali) [reply client/reducers/index.js]: sure.

File: server/server.js (hunk 7 above)
Comment (Nanitali) [reply server/server.js]: removed the conflicts

**Review by emilyparkes — APPROVED** [review]

**Pull request merged**